packages/coordinator/src/components/Table/CardTableItem.tsx

```diff
@@ -50,7 +50,7 @@ export default function CardTableItem<T>({
     createStyles({
       root: {
         // From styles/layout/grid -> $gutter-vertical-small
-        marginBottom: ".5rem",
+        margin: ".5rem 0",
```

Comment:
Nice, can you explain why this fixes the issue?

Reply:
It adds a margin on the top, 
bringing the card lower and showing it as a card with a lower and upper shadow-
instead of a divider